go.mod

```diff
@@ -1,9 +1,18 @@
 module code.cloudfoundry.org/cfhttp
 
-go 1.16
+go 1.17
```

Comment:
Perhaps it is worth mention that the PR bumps the golang version as well.

Reply:
I have added a bump to the current 1.19 in #28